### PR TITLE
Resolve bugs and clean up the TopicBarChart

### PR DESCRIPTION
**Pull request by kadamwhite**
- DOM rendering moved from D3 to React, to simplify code
- DOM simplified
- SCSS cleaned up & de-nested; absolute positioning removed

Result:

bar chart

### CODE REVIEW

**Comment by kadamwhite** [comment]
Added a transition

transition

**Comment by pbeshai** [comment]
Just looking at the gifs my thoughts are:
- Text between main controls and the bars seems weird. At the very least, should probably reproduce the emoji button inputs beneath the bar chart for quick changing.
- Ideally the full set of bars is visible on a phone screen. Seems like the text and bar heights can be reduced to help accomplish this.

**Comment by pbeshai** [comment]
How is the order of the bars currently determined? Probably something to think about...

File: src/components/TopicBarChart.js [hunk 1]
```diff
@@ -1,139 +1,56 @@
...
-
-  render() {
-    const { width } = this.props;
+    const responseCount = d3.sum(topics, d => d.count);
 
     return (
       <div className={'topic-bar-chart'}>
         <h3>Topics</h3>
-        <p>
+        {<p>
```
Comment (pbeshai) [inline src/components/TopicBarChart.js]: Why is this `<p>` in braces?

File: src/components/TopicBarChart.js [hunk 2]
```diff
@@ -1,139 +1,56 @@
...
-          }}
-          ref={(node) => { this.chart = node; }}
-        />
+          rutrum libero at, lacinia mauris.
+        </p>}
+        <div className="bar-group-container" style={{ maxWidth: `${width}px` }}>
+          <small className="note">{responseCount} total responses</small>
+          {topics.map((topic) => {
+            const percentage = topic.count ?
+              d3.format('.0%')(topic.count / responseCount) :
```
Comment (pbeshai) [inline src/components/TopicBarChart.js:54]: Seems dubious when responseCount = 0

File: src/components/TopicBarChart.js (hunk 1 above)
Comment (kadamwhite) [reply src/components/TopicBarChart.js]: Used to be commented out; thanks for the catch!

File: src/components/TopicBarChart.js (hunk 2 above)
Comment (kadamwhite) [reply src/components/TopicBarChart.js:54]: if responseCount is 0 topic count will by nature also be 0, so we'll hit the ternary

**Comment by kadamwhite** [comment]
@pbeshai What do you think about this:

emoji-key

**Comment by kadamwhite** [comment]
Got the 👍  in slack

**Pull request merged**